utils/raspberrypi/ctt/ctt_visualise.py

```diff
@@ -0,0 +1,38 @@
+# Some code that will save virtual macbeth charts that show the difference between optimised matricies and non optimised matricies
+import numpy as np
+from PIL import Image
+'''make patch 200 by 200 pixels:
+200x100 will be correct color
+100x100 will be new color, 100x100 old color
+if 50px gap between pixels, image will be 7 gaps + 6 patches wide, and 5 gaps + 4 patches high.
+  y --->
+x
+|
+|
+'''
+
+
+def visualise_macbeth_chart(mrgb, orgb, nrgb, filename):
```

Comment:
Perhaps making the variable names a little less cryptic would help the next person!